sdk/python/README.md

```diff
@@ -0,0 +1,64 @@
+# `kfp`: Kubeflow Pipelines SDK
+
+[![PyPI Package version](https://badge.fury.io/py/kfp.svg)](https://badge.fury.io/py/kfp)
+[![PyPI Python Version](https://img.shields.io/pypi/pyversions/kfp.svg)](https://pypi.org/project/kfp/)
+[![PyPI Downloads](https://img.shields.io/pypi/dm/kfp)](https://pypi.org/project/kfp/)
+[![Documentation Status](https://readthedocs.org/projects/kubeflow-pipelines/badge/?version=latest)](https://kubeflow-pipelines.readthedocs.io/en/stable/?badge=latest)
+[![Code Style](https://img.shields.io/badge/code%20style-yapf-brightgreen.svg)](https://github.com/google/yapf)
+
+Kubeflow Pipelines is a platform for building and deploying portable, scalable machine learning workflows based on Docker containers within the [Kubeflow](https://www.kubeflow.org/) project.
+
+Use Kubeflow Pipelines to compose a multi-step workflow ([pipeline](https://www.kubeflow.org/docs/components/pipelines/concepts/pipeline/)) as a [graph](https://www.kubeflow.org/docs/components/pipelines/concepts/graph/) of containerized [tasks](https://www.kubeflow.org/docs/components/pipelines/concepts/step/) using Python code and/or YAML. Then, [run](https://www.kubeflow.org/docs/components/pipelines/concepts/run/) your pipeline with specified pipeline arguments, rerun your pipeline with new arguments or data, [schedule](https://www.kubeflow.org/docs/components/pipelines/concepts/run-trigger/) your pipeline to run on a recurring basis, organize your runs into [experiments](https://www.kubeflow.org/docs/components/pipelines/concepts/experiment/), save machine learning artifacts to compliant [artifact registries](https://www.kubeflow.org/docs/components/pipelines/concepts/metadata/), and visualize it all through the [Kubeflow Dashboard](https://www.kubeflow.org/docs/components/central-dash/overview/).
```

Comment:
`using Python code and/or YAML`


Can this be rephrased as `using Python code and/or Container Spec`? It is because YAML is a general term, it might be better to say that we are expecting the definition is a ContainerOp or equivalent subject.

Reply:
I think I may still be partial to YAML over ContainerSpec. Python code and YAML both refer to the medium used to construct the pipeline definition. Put differently, users write YAML, not ContainerSpec. Additionally, the same can be said about using the term "Python code", as it is also a general term.

If you still prefer "ContainerSpec", please let me know and I can change -- just wanted to double check before updating.

Reply:
Thanks for the explanation! I understand the reason behind using a more general term for both Python and YAML here now. I am fine not making changes to the current phrasing. 

My original concern was that both Python and YAML are too generic to be understood. NIT: To help users better understanding what we are referring to, linking to this documentation is helpful https://www.kubeflow.org/docs/components/pipelines/sdk/component-development/#creating-a-component-specification. Feel free to make your decision whether to include this link or not. Thank you Connor!